README.md

```diff
@@ -132,7 +132,19 @@ end
 
 This uses ActiveJob to send notification in the background, so [make sure you configure](https://guides.rubyonrails.org/active_job_basics.html#setting-the-backend) it with the adapter you are using, if not notifications will be sent synchronously.
 
+## Async Logging
```

Comment:
We could also clarify that it's only async for requests, job errors will be logged sync